bob-common/src/stopwatch.rs

```diff
@@ -0,0 +1,121 @@
+use std::default::Default;
+use std::fmt;
+use std::time::{Duration, Instant};
+
+#[derive(Clone, Copy)]
+pub struct Stopwatch {
```

Comment:
When you'll remove `elapsed_split`, this `struct` can be transformed into 3-state `enum`:
```
pub enum Stopwatch {
    NotStarted,
    Started(Instant),
    Stopped(Duration)
}
```